### PR TITLE
Trade Improvements Path, Phase 0: Enable Download from Export lua code

### PR DESCRIPTION
**Pull request by justjuangui**
Im trying to Improve the trade feature in POB.

We should be able to download all trade information need it only once per league, not everytime.
We should be able in Export Lua app generate all files need to calculate search in POE Trade, as example:
- "Search gem in Trade"
- "Search Exact item in trade"
- "Find Optiomal item for my build in trade"
- "Calculate a aproximation of this item but make this affix not changable"
- etc....

So the first minimal change is enabled Dowload capability in the Export Lua App.

I added the script Trade.lua to show the first download as an example: "Download League information"

This code still dot use direct in POB that file but the idea is in next change be able to use that information

### CODE REVIEW

**Comment by Nightblade** [comment]
(FYI it's "Phase" not "Fase".)  😁

File: src/Export/Scripts/trade.lua [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                    out:write('\t["', league_data.id, '"] = {\n')
+                    out:write('\t\t["url"] = "', league_data.url, '",\n')
+                    out:write('\t\t["realm"] = "', league_data.realm, '",\n')
+                    out:write('\t},\n')
+                end
+            end
+            out:write('}')
+            out:close()
+        end
+    end
```
Comment (Nightblade) [inline src/Export/Scripts/trade.lua:35]: Please use TAB indentation.

**Comment by justjuangui** [comment]
> (FYI it's "Phase" not "Fase".) 😁

Sorry spanish wordd and my mind confused i will fix

File: src/Export/Scripts/trade.lua (hunk 1 above)
Comment (justjuangui) [reply src/Export/Scripts/trade.lua]: Changed

**Comment by Nostrademous** [comment]
Until all unique items in a league are found we don't know all the potential stat modifiers that exist, so `We should be able to download all trade information need it only once per league, not everytime.` can only hold true some time after the league starts. Also, that assumes there is no patch to a league mechanic/modifier stat mid-league.

**Comment by Nostrademous** [comment]
Also, calling `Download League information` only once will prevent compatibility with private leagues (which can be pulled down by PoB users that provide their POESESSID in PoB) which start mid-league.

**Pull request closed without merging**